lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb

```diff
@@ -10,6 +10,11 @@
 extend Rialto::Etl::NamedGraphs
 extend Rialto::Etl::Vocabs
 
+def contextualized_organization_name(organization)
```

Comment:
Do we want to add a special case for "Stanford University" as parent?

Reply:
If we  want to, let's make that a new ticket.